### PR TITLE
Added Heroes of the Storm

### PR DESCRIPTION
**Pull request by ImperatorS79**

### CODE REVIEW

File: Applications/Games/Heroes of the Storm/application.json [hunk 1]
```diff
@@ -0,0 +1,4 @@
+{
+	"name":	"Heroes of the Storm",
+	"description": "Heroes of the Storm (HotS) is a multiplayer online battle arena video game developed and published by Blizzard Entertainment"
```
Comment (plata) [inline Applications/Games/Heroes of the Storm/application.json]: "." missing at the end of the sentence.

File: Applications/Games/Heroes of the Storm/Online/script.json [hunk 2]
```diff
@@ -0,0 +1,7 @@
+{
+	"scriptName": "Online",
+	"compatibleOperatingSystems": ["LINUX"],
+	"testingOperatingSystems": ["LINUX"],
```
Comment (plata) [inline Applications/Games/Heroes of the Storm/Online/script.json]: Why testing?

File: Applications/Games/Heroes of the Storm/Online/script.js [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+new OnlineInstallerScript()
+    .name("Heroes of the Storm")                       
+    .editor("Blizzard")                               
+    .applicationHomepage("http://eu.battle.net/heroes/")  
+    .author("ImperatorS79")                     
+    .url("https://eu.battle.net/download/getInstaller?os=win&installer=Heroes-of-the-Storm-Setup.exe")                       
+    .category("Games")                          
+    .executable("Heroes of the Storm.exe")
+    //The checksum is different each time you download
```
Comment (madoar) [inline Applications/Games/Heroes of the Storm/Online/script.js]: @plata I'm not sure, but I think, that the checksum is currently ignored, right?
If we use the checksum later on to validate the downloaded file, we shouldn't forget to add the possibility that the checksum is unknown.

Comment (plata) [reply Applications/Games/Heroes of the Storm/Online/script.js]: Without checking, I think that the checksum is optional. So if you leave it aways, it's ignored. I think what @ImperatorS79 did is correct.

File: Applications/Games/Heroes of the Storm/Online/script.json (hunk 2 above)
Comment (ImperatorS79) [reply Applications/Games/Heroes of the Storm/Online/script.json]: I don't own the game

**Pull request merged**